src/main/scala/org/allenai/plugins/CoreRepositories.scala

```diff
@@ -61,5 +67,15 @@ object CoreRepositories {
         if (isSnapshot.value) Some(ai2PublicSnapshots) else Some(ai2PublicReleases)
       }
     }
+
+    val ai2BintrayPrivate = Seq(
```

Comment:
You may wish to comment these - something like `Settings to make bintray publishing work` or whatever. There's another setting that's needed to enable it as well, right?